src/vs/platform/windows/common/windows.ts

```diff
@@ -243,6 +243,9 @@ export interface IWindowSettings {
 	closeWhenEmpty: boolean;
 	smoothScrollingWorkaround: boolean;
 	clickThroughInactive: boolean;
+	transparent: boolean;
+	compositionAttribute: 'none' | 'transparent' | 'blur' | 'acrylic';
+	vibrancy: 'none' | 'appearance-based' | 'light' | 'dark' | 'titlebar' | 'medium-light' | 'ultra-dark';
```

Comment:
Would you think `enableVibrancy` instead of `vibrancy` would be fit here to be better consistent with @Superpencil's PR?